Auto update failing for lua-language-server

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the lua-language-server package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="bash">
INFO: Updating lua-language-server [Current version: 3.2.4]
INFO: package being updated to 3.2.5.
INFO: Trying to build package.
Running container 'termux-package-builder' from image 'termux/package-builder'...
termux - building lua-language-server for arch aarch64...
Downloading https://packages-cf.termux.org/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13133  100 13133    0     0  47756      0 --:--:-- --:--:-- --:--:-- 47756
Downloading https://packages-cf.termux.org/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   3470      0 --:--:-- --:--:-- --:--:--  3470
gpg: Signature made Thu 09 Jun 2022 06:32:44 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.com>" [ultimate]
Downloading https://packages-cf.termux.org/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13319  100 13319    0     0   149k      0 --:--:-- --:--:-- --:--:--  149k
Downloading https://packages-cf.termux.org/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   3544      0 --:--:-- --:--:-- --:--:--  3529
100   833  100   833    0     0   3529      0 --:--:-- --:--:-- --:--:--  3514
gpg: Signature made Thu 09 Jun 2022 06:56:39 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.com>" [ultimate]
Downloading https://packages-cf.termux.org/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13119  100 13119    0     0  44471      0 --:--:-- --:--:-- --:--:-- 44471
Downloading https://packages-cf.termux.org/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   3179      0 --:--:-- --:--:-- --:--:--  3179
gpg: Signature made Mon 06 Jun 2022 06:34:21 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.com>" [ultimate]
Downloading dependency libandroid-spawn@0.3 if necessary...
Found libandroid-spawn in https://packages-cf.termux.org/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.org/apt/termux-main/pool/main/liba/libandroid-spawn/libandroid-spawn_0.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 15216  100 15216    0     0  53957      0 --:--:-- --:--:-- --:--:-- 53766
extracting libandroid-spawn...
Cloning into '/home/builder/.termux-build/lua-language-server/cache/tmp-checkout'...
Note: switching to 'afb4f838f65d22443ccf30089b2f898fc9c55430'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

Submodule '3rd/EmmyLuaCodeStyle' (https://github.com/CppCXY/EmmyLuaCodeStyle) registered for path '3rd/EmmyLuaCodeStyle'
Submodule '3rd/bee.lua' (https://github.com/actboy168/bee.lua) registered for path '3rd/bee.lua'
Submodule '3rd/love-api' (https://github.com/love2d-community/love-api) registered for path '3rd/love-api'
Submodule '3rd/lovr-api' (https://github.com/bjornbytes/lovr-docs) registered for path '3rd/lovr-api'
Submodule '3rd/lpeglabel' (https://github.com/sqmedeiros/lpeglabel) registered for path '3rd/lpeglabel'
Submodule '3rd/luamake' (https://github.com/actboy168/luamake) registered for path '3rd/luamake'
Cloning into '/home/builder/.termux-build/lua-language-server/cache/tmp-checkout/3rd/EmmyLuaCodeStyle'...
Cloning into '/home/builder/.termux-build/lua-language-server/cache/tmp-checkout/3rd/bee.lua'...
Cloning into '/home/builder/.termux-build/lua-language-server/cache/tmp-checkout/3rd/love-api'...
Cloning into '/home/builder/.termux-build/lua-language-server/cache/tmp-checkout/3rd/lovr-api'...
Cloning into '/home/builder/.termux-build/lua-language-server/cache/tmp-checkout/3rd/lpeglabel'...
Cloning into '/home/builder/.termux-build/lua-language-server/cache/tmp-checkout/3rd/luamake'...
Submodule path '3rd/EmmyLuaCodeStyle': checked out '8f513f2cf3f007ea5f1240176761a408f3d343da'
Submodule path '3rd/bee.lua': checked out 'fbe9fc3c1f4730da2c0cd535ec00876af57ebda9'
Submodule path '3rd/love-api': checked out '69e7e015698c0834798a0b9ffeb3a98786b60f1d'
Submodule path '3rd/lovr-api': checked out '287070fc7e2e7b75407c4744c173497c89e1e516'
Submodule path '3rd/lpeglabel': checked out 'ed183860289664af0f3727307653c9bf2bccdc80'
Submodule path '3rd/luamake': checked out '945887aaf2c364cf12b6c92afdcf739173b59789'
Submodule '3rd/bee.lua' (https://github.com/actboy168/bee.lua) registered for path '3rd/luamake/3rd/bee.lua'
Cloning into '/home/builder/.termux-build/lua-language-server/cache/tmp-checkout/3rd/luamake/3rd/bee.lua'...
Submodule path '3rd/luamake/3rd/bee.lua': checked out '0afe33ab9945023d42c9d2b348dce02ce228e333'
Applying patch: *.patch.beforehostbuild
Downloading https://github.com/ninja-build/ninja/releases/download/v1.10.2/ninja-linux.zip
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  100k  100  100k    0     0  1275k      0 --:--:-- --:--:-- --:--:-- 1275k
Archive:  /home/builder/.termux-build/lua-language-server/tmp/ninja-1.10.2.zip
  inflating: /home/builder/.termux-build/_cache/ninja-1.10.2/ninja  
[1/29] Compile C   build/linux/obj/source_bee/lua-seri.obj
[2/29] Compile C++ build/linux/obj/source_bee/error.obj
[3/29] Compile C++ build/linux/obj/source_bee/os.obj
[4/29] Compile C++ build/linux/obj/source_bee/endpoint.obj
[5/29] Compile C++ build/linux/obj/source_bee/socket.obj
[6/29] Compile C++ build/linux/obj/source_bee/fsevent_linux.obj
[7/29] Compile C++ build/linux/obj/source_bee/simplethread_posix.obj
[8/29] Compile C++ build/linux/obj/source_bee/subprocess_posix.obj
[9/29] Compile C++ build/linux/obj/source_bee/format.obj
[10/29] Compile C++ build/linux/obj/source_bee/file_handle.obj
[11/29] Compile C++ build/linux/obj/source_bee/file_handle_posix.obj
[12/29] Compile C++ build/linux/obj/source_bee/file_handle_linux.obj
[13/29] Compile C++ build/linux/obj/source_bee/path_helper.obj
[14/29] Compile C++ build/linux/obj/source_bee/lua_platform.obj
[15/29] Compile C++ build/linux/obj/source_bee/lua_serialization.obj
[16/29] Compile C++ build/linux/obj/source_bee/lua_filesystem.obj
[17/29] Compile C++ build/linux/obj/source_bee/lua_filewatch.obj
[18/29] Compile C++ build/linux/obj/source_bee/lua_socket.obj
[19/29] Compile C++ build/linux/obj/source_bee/lua_time.obj
[20/29] Compile C   build/linux/obj/source_lua/utf8_crt.obj
[21/29] Compile C++ build/linux/obj/source_bee/lua_subprocess.obj
[22/29] Compile C++ build/linux/obj/source_bootstrap/main.obj
[23/29] Compile C++ build/linux/obj/source_bootstrap/progdir.obj
[24/29] Compile C++ build/linux/obj/source_bee/lua_thread.obj
[25/29] Compile C   build/linux/obj/source_lua/onelua.obj
[26/29] Link    Exe build/linux/bin/luamake
[27/29] Copy 3rd/bee.lua/bootstrap/main.lua build/linux/bin/main.lua
[27/29] Run test.
OS:       	Linux
Arch:     	x86_64
Compiler: 	GCC 11.2.0
CRT:      	libstdc++ 20210923 glibc 2.34
DEBUG:    	false
...................................................................................................
Ran 99 tests in 0.372 seconds, 99 successes, 0 failures
OK
[28/29] Copy build/linux/bin/luamake luamake
Applying patch: bee.lua-project-common.patch
Applying patch: loslib.patch
Applying patch: luaconf.patch
[1/87] Compile C   build/obj/source_bee/lua-seri.obj
[2/87] Compile C   build/obj/lpeglabel/lplvm.obj
[3/87] Compile C++ build/obj/source_bee/error.obj
[4/87] Compile C++ build/obj/source_bee/os.obj
[5/87] Compile C++ build/obj/source_bee/endpoint.obj
[6/87] Compile C++ build/obj/source_bee/socket.obj
[7/87] Compile C++ build/obj/source_bee/format.obj
[8/87] Compile C++ build/obj/source_bee/simplethread_posix.obj
[9/87] Compile C++ build/obj/source_bee/fsevent_linux.obj
[10/87] Compile C++ build/obj/source_bee/file_handle.obj
[11/87] Compile C++ build/obj/source_bee/subprocess_posix.obj
[12/87] Compile C++ build/obj/source_bee/file_handle_linux.obj
[13/87] Compile C++ build/obj/source_bee/file_handle_posix.obj
[14/87] Compile C++ build/obj/source_bee/path_helper.obj
[15/87] Compile C++ build/obj/source_bee/lua_platform.obj
[16/87] Compile C++ build/obj/source_bee/lua_filewatch.obj
[17/87] Compile C++ build/obj/source_bee/lua_filesystem.obj
[18/87] Compile C++ build/obj/source_bee/lua_serialization.obj
[19/87] Compile C++ build/obj/source_bee/lua_socket.obj
[20/87] Compile C++ build/obj/source_bee/lua_subprocess.obj
[21/87] Compile C   build/obj/source_lua/utf8_crt.obj
[22/87] Compile C   build/obj/source_lua/onelua.obj
FAILED: build/obj/source_lua/onelua.obj 
aarch64-linux-android-clang -MMD -MT build/obj/source_lua/onelua.obj -MF build/obj/source_lua/onelua.obj.d -std=c11 -O2 -Wall -Werror -DMAKE_LIB -DLUA_USE_LINUX -DNDEBUG  -fstack-protector-strong -Oz -DBEE_ENABLE_FILESYSTEM  -I/data/data/com.termux/files/usr/include -Wno-maybe-uninitialized -o build/obj/source_lua/onelua.obj -c 3rd/bee.lua/3rd/lua/onelua.c
error: unknown warning option '-Wno-maybe-uninitialized'; did you mean '-Wno-uninitialized'? [-Werror,-Wunknown-warning-option]
[23/87] Compile C++ build/obj/source_bee/lua_time.obj
[24/87] Compile C++ build/obj/source_bee/lua_thread.obj
ninja: build stopped: subcommand failed.
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2022-06-09 12:29:33 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/2468206213">2468206213</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/lua-language-server/build.sh
+++ b/packages/lua-language-server/build.sh
@@ -2,7 +2,7 @@ TERMUX_PKG_HOMEPAGE="https://github.com/sumneko/lua-language-server"
 TERMUX_PKG_DESCRIPTION="Sumneko Lua Language Server coded in Lua"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="Aditya Alok <alok@termux.org>"
-TERMUX_PKG_VERSION="3.2.4"
+TERMUX_PKG_VERSION=3.2.5
 TERMUX_PKG_GIT_BRANCH="${TERMUX_PKG_VERSION}"
 TERMUX_PKG_SRCURL="https://github.com/sumneko/lua-language-server.git"
 TERMUX_PKG_DEPENDS="libandroid-spawn"
@@ -14,7 +14,7 @@ _patch_on_device() {
 	if [ "${TERMUX_ON_DEVICE_BUILD}" = true ]; then
 		(
 			cd "${TERMUX_PKG_SRCDIR}"
-			patch --silent -p1 <"${TERMUX_PKG_BUILDER_DIR}"/android.diff
+			patch --silent -p1 < "${TERMUX_PKG_BUILDER_DIR}"/android.diff
 		)
 	fi
 }
@@ -31,7 +31,8 @@ termux_step_host_build() {
 }
 
 termux_step_make() {
-	CFLAGS+=" -DBEE_ENABLE_FILESYSTEM" # without this, it tries to link against its own filesystem lib and fails.
+	CFLAGS+=" -DBEE_ENABLE_FILESYSTEM"     # without this, it tries to link against its own filesystem lib and fails.
+	CFLAGS+=" -Wno-unknown-warning-option" # for -Wno-maybe-uninitialized argument.
 
 	sed \
 		-e "s%\@FLAGS\@%${CFLAGS} ${CPPFLAGS}%g" \
@@ -46,7 +47,7 @@ termux_step_make() {
 termux_step_make_install() {
 	local datadir="${TERMUX_PREFIX}/share/${TERMUX_PKG_NAME}"
 
-	cat >"${TERMUX_PREFIX}/bin/${TERMUX_PKG_NAME}" <<-EOF
+	cat > "${TERMUX_PREFIX}/bin/${TERMUX_PKG_NAME}" <<- EOF
 		#!${TERMUX_PREFIX}/bin/bash
 
 		# After action of termux-elf-cleaner lua-language-server's binary is unable to

